generic-styles/index.less

```diff
@@ -41,3 +46,6 @@
     border-spacing: 0;
   }
 }
+html {
+  -webkit-text-size-adjust: 100%; /* Prevent font scaling in landscape while allowing user zoom */
```

Comment:
this is fine, but can we investigate maybe using the viewport meta tags instead?